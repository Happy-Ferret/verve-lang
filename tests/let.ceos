let a = 1 {
  print(a)
}

expensive(): String {
  print("expensive")
  "cheap"
}

let cheap = expensive() {
  print(cheap)
  print(cheap)
}
