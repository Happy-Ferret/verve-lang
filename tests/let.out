1
expensive
cheap
cheap
